src/scss/components/lists/_list-box.scss

```diff
@@ -1,38 +0,0 @@
-@import '../../variables';
-@import '../../mixins';
-
-%ui-listbox {
```

Comment:
This is no longer needed as a `ListBox` now fully uses the `SettingsPanel` styling